### ISSUE TITLE
Question about entropy loss

### ISSUE THREAD

**Issue by yjy765** [yikun2019/PENCIL#5]
In the paper, entropy loss "helps avoid the training from being stalled in our PENCIL framework, because the label distribution is not going to be a one-hot distribution and then f(x;\theta) will be different from y^d"
In the code, entropy loss is same with the definition \Sigma -p * log p in PENCIL.py line 355.

However, when you minimize entropy loss, it "force the network to peak at only one category rather than being flat because the one-hot distribution has the smallest possible entropy value" as written in just a few phrase above.

So, I am confused if you want to minimize entropy loss or not as your intention is opposite to entropy minimization though your final loss function reduces entropy loss.
Is there anything I am wrong with?
